Scripts/World/world_manager.gd

```
@@ -75,6 +84,10 @@ func get_player_index(player : Player) -> int:
 
 
 func get_player_by_index(index : int) -> Player:
+	if index < 0:
+		return null
```

is this needed in a legitimate case or just hides errors to be fixed?
maybe a `push_warning`

---

neutrals are encoded as -1

---

i suggest we add 
`const NEUTRAL_PLAYER_INDEX = -1`
and check for it here